template/web/gatsby-config.js

```diff
@@ -7,7 +7,7 @@ const clientConfig = require('./client-config')
 
 const isProd = process.env.NODE_ENV === 'production'
 
-const SITE_URL = 'localhost:8000'
+const SITE_URL = 'https://samm-boiler.netlify.app'
```

Comment:
I think we should set this value dynamically depending on environment - example for that is in Gauvain

Reply:
Like this? But we don't have a production site atm. and for our boilerplate we never will.

```
const SITE_URL =
  process.env.STAGE === 'production'
    ? 'https://www.gauvainstudios.com/'
    : 'https://stagin--gauvain.netlify.app/';

```